src/components/LocationSelector/LocationSelector.tsx

```diff
@@ -19,7 +19,9 @@ import styles from './LocationSelector.scss';
 
 const { block, elem } = bem('LocationSelector', styles);
 
-interface Props extends LocationSelectorDialogWithGoogleLoaderProps {
+interface Props
+    extends LocationSelectorDialogWithGoogleLoaderProps,
+        Omit<React.HTMLAttributes<HTMLDivElement>, 'id'> {
```

Comment:
Do we need to omit `id` here ? It's already defined as optional string in `HTMLAttributes`

Reply:
yes, and the definitions clash, so had to Omit it.

Reply:
Interesting 🤔 I've tried to remove omit on my local and TS didn't give any error or warning

Reply:
🤔  I tried it again, and now TS is happier. Good catch. Thanks. I will remove it